adapter/internal/eventhub/subscription.go

```diff
@@ -166,7 +153,7 @@ func LoadSubscriptionData(configFile *config.Config, initialAPIUUIDListMap map[s
 	for _, configuredEnv := range configuredEnvs {
 		queryParamMap := make(map[string]string, 1)
 		queryParamMap[GatewayLabelParam] = configuredEnv
-		go InvokeService(ApisEndpoint, APIListMap[configuredEnv], queryParamMap, APIListChannel, 0)
+		go InvokeService(ApisEndpoint, apiList, queryParamMap, APIListChannel, 0)
```

Comment:
double check if having a common apiList reference for all the env based calls.

Reply:
InvokeService just uses the type of the apiList variable via reflect (to parse the response received from control plane).